src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -296,37 +412,51 @@ List of actions supported:
 `;
 
 const UPGRADE_START_OPTIONS = `
-'upgrade start' is a noobaa-cli command that will start config directory upgrade run.
-Run 'upgrade start' after upgrading NooBaa RPMs on all the cluster nodes, after starting an upgrade of the config directory,
-S3 I/O, S3 Buckets getters and NooBaa CLI Account/Buckets/Whitelist getters operations will still be working
-But updates of the config directory will be blocked during the upgrade of the config directory.
-'upgrade start' should be executed on one node, the config directory changes will be available for all the nodes of the cluster.
+Help:
+
+    'upgrade start' is a noobaa-cli command that will start config directory upgrade run.
+    Run 'upgrade start' after upgrading NooBaa RPMs on all the cluster nodes, after starting an upgrade of the config directory,
+    S3 I/O, S3 Buckets getters and NooBaa CLI Account/Buckets/Whitelist getters operations will still be working
```

Comment:
@romayalon - NooBaa CLI logging and diagnose are not available?

Reply:
fix in issue : https://github.com/noobaa/noobaa-core/issues/8595